### PR TITLE
Normalize markers formatter

### PR DESCRIPTION
**Pull request by iMoses**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/series.ts [hunk 1]
```diff
@@ -775,4 +782,53 @@ export abstract class Series<
...
+        params: TParams & Omit<AgSeriesMarkerFormatterParams<TDatum>, 'seriesId'>,
+        defaultStyle: AgSeriesMarkerStyle = marker.getStyle()
+    ) {
+        const { point } = params.datum;
+        const activeStyle = this.getMarkerStyle(marker, params, defaultStyle);
+
+        markerNode.setProperties({
+            visible: this.visible && activeStyle.size > 0 && point && !isNaN(point.x) && !isNaN(point.y),
+            translationX: point?.x,
+            translationY: point?.y,
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/series/series.ts]: Should these be present in this call, given the guard in the following statement?

File: packages/ag-charts-community/src/chart/series/seriesMarker.ts [hunk 2]
```diff
@@ -43,4 +60,15 @@ export class SeriesMarker extends ChangeDetectable {
     @Validate(OPT_NUMBER(0, 1))
     @SceneChangeDetection({ redraw: RedrawType.MAJOR })
     strokeOpacity: number = 1;
+
+    @Validate(OPT_FUNCTION)
+    // @SceneChangeDetection({ redraw: RedrawType.MAJOR })
```
Comment (alantreadway) [inline packages/ag-charts-community/src/chart/series/seriesMarker.ts]: Intended?

**Review by alantreadway — APPROVED** [review]
Couple of comments, but otherwise LGTM when addressed

File: packages/ag-charts-community/src/chart/series/seriesMarker.ts (hunk 2 above)
Comment (iMoses) [reply packages/ag-charts-community/src/chart/series/seriesMarker.ts]: No :X good catch

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**